packages/calling/src/CallingClient/calling/call.ts

```diff
@@ -2764,7 +2763,7 @@ export class Call extends Eventing<CallEventTypes> implements ICall {
    * @param localAudioTrack -.
    */
   public mute = (localAudioStream: LocalMicrophoneStream): void => {
-    if (localAudioStream) {
+    if (localAudioStream && !localAudioStream.systemMuted) {
       localAudioStream.setUserMuted(!this.muted);
       this.muted = !this.muted;
```

Comment:
Would it make sense to mute the call object when a system mute occurs? Or are we leaving this completely to the developer?

Reply:
When I mute the call object on system mute, it messes up the mute state entirely. Tried it and it was messing up the mute state.

Reply:
@Kesari3008 let's discuss this.. wouldn't we have a situation where the call object doesn't show it as mute but the call is actually muted?